webui/src/pages/auth/groups/index.tsx

```diff
@@ -97,7 +149,7 @@ const GroupsContainer = () => {
 
             <DataTable
                 results={results}
-                headers={['', 'Group ID', 'Created At']}
+                headers={['', 'Group ID', 'Permission', 'Created At', 'Repositories']}
```

Comment:
_personal opinion for your consideration (feel free to disregard)_
Assumptions: 

1. For the general use case, `Created At` is the least interesting column
2. `Repositories` is the last column to take up the remainder of the space

If these two assumptions are correct, I'd suggest moving `Created At` before `Permission` so that it's easier to quickly scan `Permission` and `Repositories` together with your eyes.
If no. 2 isn't correct, I'd put `Created At` last.

Reply:
Sure!  This is the requested design, so not touching it, sorry.

Reply:
Fair enough